Test/Microsoft.ApplicationInsights.Test/Standalone/AppInsightsStandaloneTests.cs

```diff
@@ -87,7 +86,7 @@ static void Main(string[] args)
                         var tc = new TelemetryClient();
                         using (var requestOperation = tc.StartOperation<RequestTelemetry>(""request"", args[0]))
                         {
-                            using (var dependencyOperation = tc.StartOperation<DependencyTelemetry>(""dependency""))
+                            using (var dependencyOperation = tc.StartOperation<DependencyTelemetry>(""dependency"", args[0]))
```

Comment:
can you please test both overloads.

Actually - one of the most important things for me is that `TrackTrace` will not crash due to `OperationCorrelationTelemetryInitializer` initialization failure without DS present

Reply:
The thing is, that StartOperation<DependencyTelemetry>("dependency") does not compile any longer without DS being referenced.

For .NET project, as AppInsights SDK is installed/updated, DiagnosticSource automatically by nuget.

The only problem is non .NET (non-nuget) scripts that could still use AI. As I mention here: https://github.com/Microsoft/ApplicationInsights-dotnet/issues/644#issuecomment-349754193
it's extremely unlikely that such scripts use StartOperation at all, as it's a generic method and could be called from proweshell only with reflection.

As for the StartOperation overloads: all of them except Activity-one are internally calling the same StartOperation(T telemetry), so they all use Activity underneath (when available).

I added OperationCorrelationTelemetryInitializer into the configuration and call to TrackTrace.